planner/core/planbuilder.go

```diff
@@ -1746,7 +1747,7 @@ func (b *PlanBuilder) convertValue(valueItem ast.ExprNode, mockTablePlan Logical
 			RetType: &x.Type,
 		}
 	default:
-		expr, _, err = b.rewrite(valueItem, mockTablePlan, nil, true)
+		expr, _, err = b.rewrite(context.TODO(), valueItem, mockTablePlan, nil, true)
```

Comment:
ditto

Reply:
This function is used in the web API, there is no need to use a real `ctx`.